test/distributed/tensor/test_tensor_ops.py

```diff
@@ -53,6 +53,38 @@ def test_clone(self):
             self.assertFalse(cloned_mat is mat)
             self.assertEqual(cloned_mat.to_local(), mat.to_local())
 
+    @with_comms
+    def test_copy_(self):
+        device_mesh = DeviceMesh(self.device_type, list(range(self.world_size)))
+        src_specs = [[Replicate()], [Shard(0)]]
+        src_tensor = torch.randn((12, 12))
+
+        dst_tensor = torch.zeros(12, 12)
+        dst_specs = [[Replicate()], [Shard(0)]]
+        for dst_spec, src_spec in zip(dst_specs, src_specs):
+            src_dtensor = distribute_tensor(src_tensor, device_mesh, dst_spec)
+            dst_dtensor = distribute_tensor(dst_tensor, device_mesh, src_spec)
+            dst_dtensor.copy_(src_dtensor)
+            dst_tensor.copy_(src_tensor)
+            self.assertEqual(dst_dtensor.full_tensor(), dst_tensor)
+
+    # @pytest.mark.xfail
+    # @with_comms
+    # def test_copy_broadcast(self):
```

Comment:
why this being comment out? I thought you are marking xfails

Reply:
xfail did not work haha. so i commented it. its uncommented and passing in the next PR.